README.md

```diff
@@ -177,6 +177,11 @@ $ ./flutter-client ./sample/build/linux/x64/release/bundle
 #### Note
 You need to run this program by a user who has the permission to access the input devices(/dev/input/xxx), if you use the DRM backend. Generally, it is a root user or a user who belongs to an input group.
 
+Note that `FLUTTER_DRM_DEVICE` must be set properly when you use the DRM backend. The default value is `/dev/dri/card0`.
```

Comment:
Don't use the word 'note' twice in Note section. Please change to a list. and update [readme in example](https://github.com/sony/flutter-embedded-linux/tree/master/examples/flutter-drm-backend)

Reply:
Besides, could you add an additional description, which is commands users should switch from x11 is running environment to terminal x11 doesn't be running?